data/transportation/roads-system/index.html

```diff
@@ -221,50 +216,31 @@ <h5>Related Resource</h5>
-      LRS event features.</p>
-    <p>Per UDOT policy the actual location of mileposts can vary up to 50 feet, from the measured position where
-      intersections and physical infrastructure make a more 'true' positioning impractical or undesirable.</p>
-    <p>The native spatial reference for this dataset is UTM Zone 12N, NAD83 (0.01 meter coordinate precision). The GIS
-      dataset representing the UDOT LRS is stewarded by UDOT with assistance from DTS, UGRC and local government.</p>
+    <p>The Mile Reference Posts dataset contains the location of milepost signs as they located in physical world. This collection 
+        technique is different from milepost locations that are <a href="https://data-uplan.opendata.arcgis.com/datasets/linear-measure-lm-tenth-opendata/explore">derived from a linear reference system </a>
+         (LRS). The advantage of this collection method is that the mile reference posts stay in roughly the same place over time and do not change when other sections of the road, or LRS, are realigned.</p>
+    <p>For more information on the difference between mile reference posts and linear measures (mileposts derived from an LRS), please refer to <a href="https://lrs-hub.udot.utah.gov/pages/alrs-additional-information">UDOT’s webpage</a> addressing this topic or their <a href="https://drive.google.com/file/d/14QFEDCvicSNsO13fBolRjpuYgs9RGZv_/view?usp=sharing">Reference Post vs Linear Measure</a> informational document.</p>
+    <p>Mile reference post locations are derived from images captured by <a href="https://www.mandli.com/"">Mandli Communications</a>, using on-the-ground asset collection technology. These locations were then aligned to UDOT’s linear referencing system.</p>
+    <p>Mandli Communications will update this data every 2 years under the current contract. The data served on this page was captured in 2021 and reflects what was on the ground at that time.</p>
+    <p><code>FIELDS worth noting</code></p>
+    <ul>
+      <li><code>Legend</code> - Milepost number on sign
+      <li><code>Measure</code> - Location of the milepost along the LRS
+      <li><code>Station Label</code> - Highway name + direction of route along LRS (P = positive, milepost numbers increasing; N = negative, milepost numbers decreasing) + milepost number. For example the value 0015PM_271 translates to mile marker 217 on Interstate-15 on the north bound (P) side.
+    </ul>
+    <p>The Mile Reference Posts is an <a href="https://www.esri.com/en-us/arcgis/products/arcgis-roads-highways/overview">ESRI Roads and Highways Event</a> layer and supports the <a href="https://maps.udot.utah.gov/wadocuments/Apps/Stationing/">RP Stationing App</a>.</p>
+    <h5>Related Resource</h5>
+    <ul class="dotless">
+      <li><a href="https://data-uplan.opendata.arcgis.com/datasets/linear-measure-lm-tenth-opendata/explore">Linear-measure based mileposts</a> (derived from UDOT’s linear referencing system and contains tenth-mile points)
```

Comment:
**suggestion**: should we add a link to the web api's milepost geocoding here?

Reply:
good call.

Reply:
what's the best url
https://api.mapserv.utah.gov/  ?

Reply:
https://api.mapserv.utah.gov/#geocoding